### PR TITLE
Resolving Issue #33

### PR DESCRIPTION
**Pull request by Maxoudela**
I also fixed two tests failing due to my precedent commit that set a minimum size of 16 to today Button.

### CODE REVIEW

File: jfxtras-controls/src/test/java/jfxtras/scene/control/test/CalendarPickerTest.java [hunk 1]
```diff
@@ -46,363 +46,363 @@
...
-		moveBy(100,0);		
-		release(MouseButton.PRIMARY);
-		Assert.assertEquals("2013-01-01T20:34:56.000", TestUtil.quickFormatCalendarAsDateTime(calendarPicker.getCalendar()));
-		
-		// move the minute slider
-		move("#minuteSlider > .thumb");
-		press(MouseButton.PRIMARY);
-		moveBy(-50,0);		
-		release(MouseButton.PRIMARY);
-		Assert.assertEquals("2013-01-01T20:23:56.000", TestUtil.quickFormatCalendarAsDateTime(calendarPicker.getCalendar()));
```
Comment (Maxoudela) [inline jfxtras-controls/src/test/java/jfxtras/scene/control/test/CalendarPickerTest.java:319]: Don't know why GitHub is telling that I have touched everything..
Actually I've just modified that "23" to "22" because apparently, moving the thumb by 50 pixels is doing 22 minutes.

Same on the other test

**Comment by tbee** [comment]
Thanks, it works great.

I did changed the test, I felt they were not scripting enough (did not tell a story). And added two for the highlight lists.

**Comment by Maxoudela** [comment]
Yes I saw no problem it's better!

Also I noticed that they were no name in the copyright, like "<copyright Holder>" was written. Shouldn't it be replace by Jfxtras somehow?

Also, do you want me to try to implement an abstract class for factorizing some property?

**Comment by tbee** [comment]
There is a Gradle plugin to put copyright messages in the source code, someone added that quite some time ago. Need to check what is going on with that.

An abstract superclass reduces code duplication, but also forces all three controls to have the same API. It is a question of how you look at them; are they three separate controls who happen to have a similar API (loose coupling), or are they three implementations of the same abstract? Not sure what is wise there. You could do a refactor and see what problems come up and how much is gained. We can always untangle in the future if it turns out loose coupling is better.

**Comment by Maxoudela** [comment]
Well they will have the same API at first with their basic features (locale, allowNull).

Other things that annoyed me was the fact that they shared the same methods but each name was different :
-disabledLocalDates
-disabledCalendars
-disabledLocalDateTime

I would like to have a single method name so that when I implement a RangeCallBack, I can copy the method and paste it in another component without renaming everything. 

But again, this is my personal feeling, maybe it's confusing to have the same API in several components parametrized with different implementations of date.

**Comment by miho** [comment]
@tbee 

> There is a Gradle plugin to put copyright messages in the source code, someone added that quite some time ago. Need to check what is going on with that.

If I remember correctly, I only added it to the labs project. The license template .txt has been added to jfxtras though.

**Comment by tbee** [comment]
Added the license text part to the gradle.build of JFXtras and let it run. All good now.
